<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>puissance4</class>
 <widget class="QMainWindow" name="puissance4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>puissance4</string>
  </property>
  <widget class="QWidget" name="LeTout">
   <widget class="QPushButton" name="PlayPB">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>63</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Jouer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResultPB">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Résultat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SettingsPB">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Paramétres</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QuitPB">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
